EntAppProject/JBForgotPassword.html

```diff
@@ -1,43 +1,25 @@
-<!DOCTYPE html>
-
-<html>
-
-<head>
-
-<title>Forgot Username/Password?</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
-
-</head>
-
-<body> 
-
-<h1><span>Job</span>Book</h1>
-
-<h2>Forgot Username/Password?</h2>
- 
-<form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JobBookWelcomeScreen.html">
-
-Email Linked to Account:     
-<input type="email" name="username">
-<br>
-<br>
- 
-New Password:                     
-<input type="password" name="password">
-
-<br>
-
-Confirm New Password:      
-<input type="password" name="password">
-<br>
-<br>
-                
-<input type="submit" value="Submit">
-</form>
-        
-<h4>Thanks!  Instruction on how to reset your password have been sent to your email.</h4>
-  
-</body>
-
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+        <title>Forgot Username/Password?</title>
+        <link rel="stylesheet" href="JobBookCSS.css">
+    </head>
+
+    <body>
+        <h1><span>Job</span>Book</h1>
+        <h2>Forgot Username/Password?</h2>
+        <form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JobBookWelcomeScreen.html">
+            Email Linked to Account:
+            <input type="email" name="username"><br><br>
```

Comment:
A lot of the spacing in the HTML forms are created using the `<br>` tag. This works fine, but it can be a pain to maintain/resolve issues. I have two suggestions you may want to consider: 1. create a class in your stylesheet for adding padding to the bottom of an html page. Then add `class="PADDING_CLASS"` to any markup that you usually would follow with a `<br>`. 2. If you implement bootstrap into the project, you can use any of the pre-existing bootstrap padding classes. The way it works is bootstrap has precreated a series of padding classes that range from 5px to some greater number of pixels usually by increments of 5. If you want to add 10px of bottom padding to a markup, you can just add `class="p-t-10"` and it will get 10px of bottom padding.

Reply:
Nice suggestion.